php/WP_CLI/Runner.php

```diff
@@ -1989,4 +1975,43 @@ private function enable_error_reporting() {
 		}
 		ini_set( 'display_errors', 'stderr' ); // phpcs:ignore WordPress.PHP.IniSet.display_errors_Disallowed
 	}
+
+	/**
+	 * Check if the given name is a registered taxonomy.
+	 *
+	 * @param string $name The taxonomy name.
+	 * @return bool True if the taxonomy exists, false otherwise.
+	 */
+	private function is_taxonomy( $name ) {
+		// Ensure WordPress is loaded.
+		$this->load_wordpress();
+
+		// Check again after loading WordPress.
+		if ( function_exists( 'get_taxonomies' ) ) {
```

Comment:
`load_wordpress()` errors if WordPress couldn't load, so this extra check here is superfluous